tests/store/artifact/utils/test_model_utils.py

```diff
@@ -60,7 +62,6 @@ def test_parse_models_uri_with_latest(uri, expected_name):
         "notmodels:/NameOfModel/StageName",  # wrong scheme with stage
         "models:/",  # no model name
         "models:/Name/Stage/0",  # too many specifiers
-        "models:/Name/production",  # should be 'Production'
         "models:/Name/LATEST",  # not lower case 'latest'
```

Comment:
Should `"models:/Name/LATEST"` be a valid model URI or not?

Reply:
I'm leaning towards yes. Do you think we should make "latest" case insensitive as well? I can make the change.

Reply:
> I'm leaning towards yes.

I am too :)

Reply:
Thanks! Just added the commit

Reply:
thanks!